Data/Sys/GameSettings/GSR.ini# RUUE01, RUUJ01, RUUK01, RUUP01 - Animal Crossing Wii [Core] # Values set here will override the main Dolphin settings. [EmuState] # The Emulation State. 1 is worst, 5 is best, 0 is not set. EmulationStateId = 4 EmulationIssues =  [OnLoad] # Add memory patches to be loaded once on boot here. [OnFrame] [ActionReplay] [Video_Hacks] EFBToTextureEnable = False [Video_Stereoscopy] StereoConvergenceMinimum = 381

```diff
@@ -0,0 +1,14 @@
+# GSRE7S - Smuggler's Run: Warzones
+[Core]
+# Values set here will override the main Dolphin settings.
+[EmuState]
+# The Emulation State. 1 is worst, 5 is best, 0 is not set.
+EmulationStateId = 
```

Comment:
[According to our wiki](https://wiki.dolphin-emu.org/index.php?title=Smuggler%27s_Run:_Warzones) this game should have an Emulation state of 4.